packages/augur-ui/src/assets/styles/_colors.less

```diff
@@ -148,7 +148,9 @@
   --color-primary-text-40: fade(#0E0E21, 40%);
   --color-primary-text-50: fade(#0E0E21, 50%);
   --color-primary-text-80: fade(#0E0E21, 80%);
+  --color-primary-text-85: fade(#0E0E21, 85%);
```

Comment:
you gotta define these for trading (aka just `:root`) as well otherwise this will cause an error when it can't find this variable name defined.